# Message for Hybrid reference path signal.

Header header

float64 w_ref
float64 v_ref
float64 dt_v
float64 dtheta_v

geometry_msgs/Pose2D eta_d
geometry_msgs/Pose2D dot_eta_d
geometry_msgs/Pose2D ddot_eta_d